src/App.css

```diff
@@ -35,18 +36,123 @@ nav ul li a {
   font-weight: bold;
 }
 
+.book-list {
+  display: flex;
+  flex-direction: column;
+  padding: 0 5%;
+}
```

Comment:
- [ ] Kindly use the correct font family for the app as required. you are using `courier new` instead of the assigned fonts